1063

```diff
@@ -0,0 +1,26 @@
+ECIP: <1063>
+Title: <Ethereum Classic Integrated Decentralized Oracles and Smart Contracts>
+Author: <Jesse>
+Discussions-To: <email address>
+Status: <Draft>
+Type: <Standards Track | Informational | Process>
+Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+Replaces: <ECIP number>
+Superseded-By: <ECIP number>
+Resolution: <url>
+---
+# Abstract
+Imagine how we can use  ETC having oracle features which can assist Dapps and ERC-223 tokens to accomplish their goals. Although decentralized oracles rely on a source of information, it does not have to be from a centralized source. If an oracle is able to rely on multiple sources and is able to verify which information is the correct one, wouldn't it still work in a manner that is decentralized as possibly can? An example would be academia sources for a research paper, we have multiple sources that cannot be used for reliable information such as journals and memoirs.  However, with a decentralized oracle, it can then verify the reliability of an information and check if the sources have a track history of reliability such as from an academic institute. Information that is considered unreliable such as a  journal or memoir can be verified to be reliable if the journalist has a history of accuracy and reliability.  If the person who wrote the memoir is the primary source and has the evidence through other reliable sources that he is there, then the oracle can consider his memoir as a reliable source of information. Afterwards, then feed it to the smart contract to come with a reliable answer of what actually happened
```

Comment:
Please break down sentences into several lines. It's not user friendly to scroll sideways to read your content when reviewing it.